src/main/java/com/worldpay/sdk/util/WorldpayLibraryConstants.java

```diff
@@ -70,13 +70,15 @@ private WorldpayLibraryConstants() {
     public static final String LANG_VERSION_PROP =
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/nitintiwariwp/worldpay-lib-java/file/1602816680/issues/source?bid=3026869&fileBranchId=3027900#l70)